lib/colorado_lottery.rb

```diff
@@ -0,0 +1,21 @@
+class ColoradoLottery
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants
+
+  def initialize
+    @registered_contestants = ({})
+    @winners = []
+    @current_contestants = ({})
+  end
+
+  def interested_and_18?(contestant, game)
```

Comment:
Because you set up attributes and attr_readers for the contestant, we can access them with `contestant.age` when they are passed in. For the second condition of the `&&` we needed `contestant.game_interests.include?(game.name)` or `contestant.game_interests.find { |interest| interest == game.name }`. When nothing is found with `find` it returns a nil and when it finds something it returns the value it finds - which is truthy.